src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts

```diff
@@ -58,6 +57,33 @@ export class EditCellAction extends ToggleableAction {
 	}
 }
 
+export class SplitCellAction extends CellActionBase {
```

Comment:
Please add tests in cellToolbarActions.test.ts (we have coverage for other actions in this file)

Reply:
Will be adding tests in a separate task https://github.com/microsoft/azuredatastudio/issues/17267